Makefile

```diff
@@ -142,10 +142,10 @@ clean-cache:
 	$(MAKE) clean
 
 bundle: ## Installs dependencies using bundler. Run this after you make some changes to Gemfile.
-bundle: Gemfile Gemfile.on_prem
+bundle: Gemfile.prod Gemfile
```

Comment:
@josemigallas i think if we just remove `Gemfile.prod` from this line you should be all set... could you give that a go?

Reply:
Still failing